### DISCUSSION TITLE
assign_impedance_to_sheet gives unjust False return

### DISCUSSION THREAD

**Discussion by LarsWillocx** [ansys/pyaedt#5371] [Q&A] [answered]
When I use assign_impedance_to_sheet I get a False return with no further error messages. I created the design with Eigenmode solution type.

It does work for the regular solution type, but I need the Eigenmode and I don't know how to make the boundary first and then change the solution type of the design. I know there are functions for changing the solution type, but I couldn't get it to work.

When I make the boundary manually in the ansys program, it works perfectly with no error messages whatsoever.

![edit](https://github.com/user-attachments/assets/0060d40c-7532-44ac-8709-8e9f3c10f51d)

**Comment by gmalinve** [comment, marked as answer]
Hi @LarsWillocx,

Thanks for reporting this.
We open an issue and fix this in a PR that has already been merged: #5374.
It will be available in the next release.

Kind regards,

Giulia
